src/osd/OSD.cc

```diff
@@ -1660,6 +1660,13 @@ bool OSD::asok_command(string command, cmdmap_t& cmdmap, string format,
     } else {
       op_tracker.dump_ops_in_flight(f);
     }
+  } else if (command == "dump_blocked_ops") {
+    if (!op_tracker.tracking_enabled) {
+      ss << "op_tracker tracking is not enabled";
```

Comment:
we could print more warning messages instead of merely "op_tracker tracking is not enabled"? something like:

> op_tracker tracking is not enabled now, so no ops are tracked currently, even those get stuck. Please enable "osd_enable_op_tracker", and the tracker will start to track ops issued afterwards.